utils.py

```diff
@@ -475,6 +477,36 @@ def get_human_readable_time_string(time_msec):
         '%B %d %H:%M:%S', time.gmtime(python_utils.divide(time_msec, 1000.0)))
 
 
+def create_string_from_largest_unit_in_timedelta(timedelta_obj):
+    """Given the timedelta object, find the largest non zero time unit and
```

Comment:
It's worth clarifying that this returns a human-readable string, and that the string is not localized (it's time format & language is kept US-specific).